ruby_02-web_applications_with_ruby/outlines/route_helpers.md

```diff
@@ -9,66 +9,87 @@ tags: routes, helpers
 
 By the end of this lesson, you will know/be able to:
 
-  * Run a command that can find all the information about your routes - know what 5 pieces of information this tool gives us.
-  * Use a link helper to make writing html links easier.
+  * Run `rake routes` to find information about your routes.
+  * Understand the 5 pieces of information this `rake routes` gives us.
   * Use a route helper to easily refer to a relative and absolute path.
+  * Understand the difference between what `_url` and `_path` return when combined with a routes prefix.
+  * Use a link helper to make writing html links easier.
   * Find a routes prefix and use that prefix to build a helper.
 
-### What is CRUD again?
+### Review
+
+What is CRUD again?
 
   * Create
   * Read
   * Update
   * Delete
 
-### What are the specific CRUD routes I get:
+### Warmup
 
-  * If I am modeling the entity `:song`?
-  * If I am modeling the entity `:phone`?
-  * If I am modeling the entity `:product`?
-  * If I am modeling the entity `:sticker`?
+In pairs, one of you write the CRUD routes for `song`, one for `phone`. If you're in a group of three, have the third person write the routes for `sticker`.
 
-##### Do you notice any repetition or patterns here?
+With your partner, do you notice any repetition or patterns in the routes that you wrote?
 
-### Let's go practice hands on
+Large group share.
+
+### Practice
 
 #### Routes
 
-  Clone down [this repo](https://github.com/Carmer/helpers_playground).
+Clone down [this repo](https://github.com/Carmer/helpers_playground).
 
   `git clone https://github.com/Carmer/helpers_playground`
 
-  Now go to the `routes.rb` file.
+Now go to the `routes.rb` file.
+
+Let's create a `resources` line for each of the entities below and then run `rake routes` from the terminal:
+
+  * Song
+  * Phone
+  * Sticker
 
-  Let's create a `resources` for each of the entities below and look at the `rake routes`:
-  
-    * Song
-    * Phone
-    * Product
-    * Sticker
+`rake routes` gives you output in tabular format.
 
-What is a prefix?
-What is a URI pattern?
-What 5 things to we get from rake routes?
+* What are the headers of the table that it gives you?
+* With your partner, take a look at the entries in the table that `rake routes` gives you and fill out the table below in your notebook or on your computer.
+
+|`rake routes` Table Heading|Exampe Entry|Potential Definition|
```

Comment:
This might be a tad confusing since you've transposed the columns a bit (I could be completely wrong though), but I really like this activity of having them diagram this info